package.json

```diff
@@ -35,7 +35,9 @@
     "typescript": "~4.7.4"
   },
   "resolutions": {
-    "eslint-plugin-jsx-a11y": "6.7.1"
+    "eslint-plugin-jsx-a11y": "6.7.1",
+    "@types/react-native/@types/react": "17.0.34",
```

Comment:
Can you, please, explain the problem here? How does it fix security vulnerabilities? If it's something related to react-native, should it be under _sitecore-jss-react-native_ package?

Reply:
The reason for this changes is that the direct dependency @types/react-native was installing its own dependency @types/react at ver 18 and this caused type discrepancies and build failing

Reply:
My concern here, is that you are adding it to _package.json_ in the root (it's a monorepo config), so for the end customer, it doesn't change anything, it will only affect local development

Reply:
right. i added overrides in react-native's package.json

Reply:
@yavorsk Should it be removed from here in a such case?